src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -463,45 +463,8 @@ class AgoraBusiness(permissionsDataSource: PermissionsDataSource)(implicit ec: E
     findSingle(entity.namespace.get, entity.name.get, entity.snapshotId.get, entityTypes, username)
   }
 
-//  private def validateDockerImage(task: Task) = {
-//    // Per DSDEEPB-2525, in the interests of expediency, we are disabling docker image validation as we do not support validation of private docker images
-//    // When that functionality is added back in, then this is where it should go.
-//
-////    if (task.runtimeAttributes.docker.isDefined) {
-////      val dockerImageReference = parseDockerString(task.runtimeAttributes.docker.get)
-////      if (dockerImageReference.isDefined) {
-////        DockerHubClient.doesDockerImageExist(dockerImageReference.get)
-////      }
-////    }
-//    true
-//  }
-
```

Comment:
Minor nit - I understand the deletion here, but I would prefer that happen not as part of the migration to 2.12 and instead as something else. Same for the deleted block below this.

Reply:
Understood. They're back in now.